ZConfig/components/logger/tests/test_logger.py

```diff
@@ -27,11 +27,8 @@
 from ZConfig.components.logger import handlers
 from ZConfig.components.logger import loghandler
 
-try:
-    import StringIO as StringIO
-except ImportError:
-    # Python 3 support.
-    import io as StringIO
+from ZConfig._compat import NStringIO as StringIO
```

Comment:
Why not leave the name `NStringIO`, since we have to munge all the callers anyway?